docs/reference-capabilities/default-capabilities.md

```diff
@@ -0,0 +1,233 @@
+
+We go into detail of the default types below, but here's a table summarizing them:
+
+| Item                    | Default Capability | Explanation                                                             |
+| ----------------------- | ------------------ | ----------------------------------------------------------------------- |
+| primitive               | `val`              | Primitives are immutable and can never be anything but `val`            |
+| actor                   | `tag`              | Actors can neither be read nor written and are always `tag`             |
+| class                   | `ref`              | Instances of classes default to ref type if unspecified                 |
+| fields                  | default for type   | Whatever was specified on the _target_ class                            |
+| variables               | default for type   | Whatever was specified on the _target_ class                            |
+| parameter               | default for type   | Whatever was specified on the _target_ class                            |
+| constructor (primitive) | `val`              | Only classes can have different constructor capabilities                |
+| constructor (actor)     | `tag`              | Only classes can have different constructor capabilities                |
+| constructor (class)     | `ref`              | Only classes can have different constructor capabilities                |
+| receiver (function)     | `box`              | Functions cannot mutate class state by default (use `func ref`)         |
+| receiver (behaviour)    | `ref`              | Actors can mutate their state from inside a behaviour                   |
+| recover (immutable)     | `val`              | An immutable value (box or val) is recovered to `val` by default        |
+| recover (mutable)       | `iso`              | An immutable value (box or val) is recovered to `val` by default        |
+| literal (object)        | `ref`              | object literals default to same capabilities as their non-literal types |
+| literal (actor)         | `tag`              | object literals default to same capabilities as their non-literal types |
+| literal (primitive)     | `val`              | object literals default to same capabilities as their non-literal types |
+| lambda (no captures)    | `val`              | lambda capabilities depend on whether they capture their environment    |
+| lambda (captures)       | `ref`              | lambda capabilities depend on whether they capture their environment    |
+
+## Primitive default capabilities
+
+Let's start with primitives because they're easy. All primitives have `val` capabilities. They are immutable, they are read only, there is only one of each primitive, and they are accessible from all actors. So `val`.
+
+In fact, defining a primitive as `primitive val MyPrimitive` is a syntax error. Primitives are so `val` that you can't even tell them they are `val`.
```

Comment:
```suggestion
In fact, defining a primitive as `primitive val MyPrimitive` is a syntax error.
```

I like the second sentence as it brings in some personality to the Tutorial, but nonetheless I think it is not needed.